R/readIEA_Energy_Projections_Extended_Indicators.R

```diff
@@ -0,0 +1,54 @@
+#' readIEA_Energy_Projections_Extended_Indicators
+#'
+#' Read in IEA_Energy_Projections_Extended_Indicators from International Energy Agency.
+#'
+#' @param subtype Type of data that should be read.
+#' @return The read-in data into a magpie object.
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IEA_Energy_Projections_Extended_Indicators", subtype = "TES")
```

Comment:
object returned contains a typo ("varius")
also, unclear from the description what the scenarios and "extended indicators" exactly are

Reply:
the unit is in the flow, for example Total energy supply (TJ)
if it is written TES does not have units so the unit column is varius
should I use the long name which has the units?

Reply:
yes, using the long name makes more sense

Reply:
ok